browser/ui/views/frame/split_view/brave_contents_container_view.h

```diff
@@ -35,6 +35,11 @@ class BraveContentsContainerView :
   explicit BraveContentsContainerView(BrowserView* browser_view);
   ~BraveContentsContainerView() override;
 
+  void set_for_web_panel(bool for_web_panel) { for_web_panel_ = for_web_panel; }
```

Comment:
Can we do this in constructor?

```
BraveContentsContainerView(BrowserView* browser_view, bool for_web_panel = false)
```

Then we could make the member variable const indicating that it won't be changed during the lifecycle of this instance.

```
const bool for_web_panel_
```

Reply:
Much better. done!